.github/workflows/deploy-prod.yml

```diff
@@ -0,0 +1,51 @@
+on:
+  # Enables manually triggers
+  workflow_dispatch:
+  push:
+    branches:
+      - "radix-prod-env"
+    tags:
+      - "v**"
+
+env:
+  API_IMAGE: mariner.azurecr.io/for/api:latest
+  WEB_IMAGE: mariner.azurecr.io/for/web:latest
+
+jobs:
+  build-and-publish:
+    name: Build and deploy
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@master
+      - name: "Docker Login"
+        run: echo ${{ secrets.DOCKER_PASSWORD }} | docker login mariner.azurecr.io --password-stdin --username ${{ secrets.DOCKER_USERNAME }}
+      - name: Pull latest
+        run: docker pull ${API_IMAGE} && docker pull ${WEB_IMAGE}
```

Comment:
This will not work. As no image with that tag exists at this stage. Same for the other commands using the $?_IMAGE, like --cache-from